test/test_transformers.py

```diff
@@ -137,6 +137,14 @@ def _size(i):
         size = (batch, seq_len, num_heads, head_dim) if not packed else (batch, seq_len, 3 * num_heads * head_dim)
         return torch.randn(size, device=device, dtype=dtype, requires_grad=requires_grad)
 
+def calculate_nt_tolerances(nt_ref_hp, nt_ref_lp, default_dtype, fudge_factor=1):
+    # TODO use NT ops when we have implemented Max for NestedTensor instead of unrolling
+    ref_atol = default_atol[default_dtype]
+    ref_rtol = default_rtol[default_dtype]
+    for tensor_component_ref, tensor_component_ref_lp in zip(nt_ref_hp.unbind(), nt_ref_lp):
```

Comment:
nt_ref_lp.unbind()*

Reply:
`unbind()` happens implicitly when you foreach over it, no?